### PR TITLE
[V2] tableView switch to blueprintjs.

### PR DESCRIPTION
**Pull request by thealmightyone**

### CODE REVIEW

File: src/components/Table/TableToolbarComponent.jsx [hunk 1]
```diff
@@ -53,40 +47,44 @@ class TableToolbarComponent extends Component {
...
-          <TextField floatingLabelText={'Search'} value={this.state.filterValue}
-            onChange={this.handleSearchChange}
-          />
-        </ToolbarGroup>
-      </Toolbar>
+      <div>
+        <div className="pt-navbar-group pt-align-left">
+          <button type="button" className="pt-button pt-intent-primary pt-icon-add"
+            onClick={this.props.handleOpenModal}>{'New ' + this.props.singular}
+          </button>
```
Comment (sealionkat) [inline src/components/Table/TableToolbarComponent.jsx]: I recommend using `Button` component: http://blueprintjs.com/docs/#components.button
I am using it in auth components - so there will be more coherent.

File: src/components/Table/TableToolbarComponent.jsx [hunk 2]
```diff
@@ -53,40 +47,44 @@ class TableToolbarComponent extends Component {
...
+              <select value={this.state.filterProperty}
+                onChange={this.handleMenuItemSelected}>
+                {this.buildSelectOptions()}
+              </select>
+            </div>
+            <div className="pt-input-group">
+              <span className="pt-icon pt-icon-search"/>
+              <input type="text" className="pt-input"
+                placeholder="Search"
+                onChange={this.handleSearchChange}
```
Comment (sealionkat) [inline src/components/Table/TableToolbarComponent.jsx]: You can put this attribute in the same line as `placeholder` unless the line is longer than 120

File: src/components/Table/TableHeaderComponent.jsx [hunk 3]
```diff
@@ -1,13 +1,4 @@
...
-  TableRow,
-  TableHeaderColumn
-} from 'material-ui';
-
-const columnStyle = {
-  wordWrap: 'break-word',
-  whiteSpace: 'normal'
-};
 
 class TableHeaderComponent extends Component {
```
Comment (stepek) [inline src/components/Table/TableHeaderComponent.jsx]: you can remove Component suffix from class name and file name

File: src/components/Table/Table.jsx [hunk 4]
```diff
@@ -1,20 +1,10 @@
...
 import TableToolbarComponent from './TableToolbarComponent';
 import TableHeaderComponent from './TableHeaderComponent.jsx';
 import TableRowComponent from './TableRowComponent.jsx';
 import Breadcrumb from '../Breadcrumb';
 
-const detailStyle = {
-  padding: 15
-};
-
 class TableComponent extends Component {
```
Comment (stepek) [inline src/components/Table/Table.jsx]: you can remove Component suffix from class name.

File: src/components/Table/TableRowComponent.jsx [hunk 5]
```diff
@@ -1,22 +1,7 @@
...
-  wordWrap: 'break-word',
-  whiteSpace: 'normal'
-};
-
-const iconStyle = {
-  color: 'rgba(0, 0, 0, 0.55)'
-};
+import {Tooltip} from '@blueprintjs/core';
 
 class TableRowComponent extends Component {
```
Comment (stepek) [inline src/components/Table/TableRowComponent.jsx]: you can remove Component suffix from class name and file name.

File: src/components/Table/TableToolbarComponent.jsx [hunk 6]
```diff
@@ -1,10 +1,5 @@
 import React, {Component} from 'react';
-import {Toolbar, ToolbarGroup, RaisedButton, SelectField, TextField, MenuItem} from 'material-ui';
-
-const raisedButtonStyle = {
-  margin: '8px 8px 8px 8px',
-  marginRight: 'auto'
-};
+import {Button} from '@blueprintjs/core';
 
 class TableToolbarComponent extends Component {
```
Comment (stepek) [inline src/components/Table/TableToolbarComponent.jsx]: you can remove Component suffix from class name and file name

File: src/components/Table/TableToolbarComponent.jsx [hunk 7]
```diff
@@ -53,40 +48,44 @@ class TableToolbarComponent extends Component {
...
-          />
-        </ToolbarGroup>
-        <ToolbarGroup>
-          <SelectField floatingLabelText={'Filter by'} value={this.state.filterProperty}
-            onChange={this.handleMenuItemSelected}>
-            {this.buildSelectOptions()}
-          </SelectField>
-          <TextField floatingLabelText={'Search'} value={this.state.filterValue}
-            onChange={this.handleSearchChange}
+      <div>
```
Comment (stepek) [inline src/components/Table/TableToolbarComponent.jsx]: why you close div in other div?

Comment (thealmightyone) [inline src/components/Table/TableToolbarComponent.jsx]: I'm nore sure what you're refering to here. Can you elaborate?

Comment (sealionkat) [inline src/components/Table/TableToolbarComponent.jsx]: @stepek There is a list of divs in this one div.

File: src/components/Table/Table.jsx [hunk 8]
```diff
@@ -31,35 +21,35 @@ class TableComponent extends Component {
...
+      <div className={'pt-card pt-elevation-3'}>
         <Breadcrumb breadcrumbs={[singular]}/>
         <TableToolbarComponent handleOpenModal={this.props.openModal} singular={singular}
           options={schema.propertiesOrder} properties={schema.properties}
           filterData={this.props.filterData} filterProperties={this.props.visibleColumns}
         />
 
-        <Table fixedHeader={true}>
-          <TableHeader>
+        <table className={'pt-table pt-interactive'} style={{width: '100%'}}>
```
Comment (stepek) [inline src/components/Table/Table.jsx]: Please don't use inline styles.

File: src/components/Table/TableToolbarComponent.jsx (hunk 7 above)
Comment (stepek) [inline src/components/Table/TableToolbarComponent.jsx]: ahh ok thanks

**Comment by stepek** [comment]
Please fix comments

**Comment by thealmightyone** [comment]
Should be okay now!

**Pull request merged**